service-config/build.gradle

```diff
@@ -0,0 +1,13 @@
+apply from: "$rootDir/gradle/libs.gradle"
+apply from: "$rootDir/gradle/publish.gradle"
+
+dependencies {
+    compile project(':http-clients')
```

Comment:
updated. Original thought was that consuming apps just need to pull in service config and not to worry about feign client or how to build the it.